dwitter/views_v2.py

```diff
@@ -70,18 +71,30 @@ class DweetViewSet(mixins.RetrieveModelMixin,
     queryset = Dweet.objects.all().select_related(
         'author',
     ).prefetch_related(
+        Prefetch('reply_to', queryset=Dweet.objects.select_related('author')),
```

Comment:
This one should probably be in select related instead of prefetch, since it is a forward fk.

```python
.select_related(
  ..,
  'reply_to',
  'reply_to__author'
)

Reply:
Oh cool, I was looking for a way to do that :)

Reply:
For sanity I checked:
Removing the prefetch goes from 8 to 19 queries.
Adding 'reply_to' to select_related brings it down to 13, and 'reply_to__author' further down to 7 (!)